.travis.yml

```diff
@@ -45,6 +45,9 @@ before_script:
   - cat nndc_xs/nndc.tar.gza* | tar xzvf -
   - rm -rf nndc_xs
   - export OPENMC_CROSS_SECTIONS=$PWD/nndc/cross_sections.xml
+  - wget http://web.mit.edu/smharper/Public/multipole_lib.tar.gz
```

Comment:
I feel we should have this stored somewhere a little more permanent. We have NNDC binaries stored in @bhermanmit's nndc_xs github repo; can we just create a github repo for this too?